CONTRIBUTING.md

```diff
@@ -113,6 +113,32 @@ If you'd like to contribute example `.bicep` files that showcase abilities of th
 
 **Note:** If you have never submitted a Pull Request or used git before, reading through the [Git tutorial](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/git-tutorial.md) in the azure-quickstart-template repo is a good place to start.
 
+### Snippets
+
+If you'd like to contribute to the collection of snippets:  
+
+* A snippet should be a single, generic, resource. Ensure your snippet meets this criteria.
+* Add a Bicep file here, using the naming convention `res-RESOURCENAME.bicep`: [`./src/Bicep.LangServer/Snippets/Templates`](./src/Bicep.LangServer/Snippets/Templates)
+  * The file name without extension will be used as the label.
+  * A single line comment at the top of the file will be used as the description.
+  * E.g. [`res-aks-cluster.bicep`](./src/Bicep.LangServer/Snippets/Templates/res-aks-cluster.bicep) results in the following label and description:
+ ![](./docs/images/snippet-template.png)
+
+* Add a new folder in the following directory, for an integration test that validates snippet completion: [`./src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates`](./src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates)
+  * The folder name should match the snippet label/prefix.
+
+* Add a file named main.bicep
+  * The test will read this input file for snippet completions. It will take the replacement values listed in this file, replace them with their corresponding placeholder, then ensure Bicep reports no warnings or errors.
+
+  * E.g. [`res-aks-cluster/main.bicep`](./src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates/res-aks-cluster/main.bicep)
+
+* Add a file named diagnostics.json - this should contain information about expected errors. There should not be any for new contributions, this is used for things like deprecated features.
+  * E.g. [`res-aks-cluster/diagnostics.json`](./src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates/res-aks-cluster/diagnostics.json)
+
+* Submit a PR for review
+
```

Comment:
Might want to encourage them to run `dotnet test` to ensure their tests are passing

Reply:
Added